laterpay/asset_sources/js/blocks/contribution/index.js

```diff
@@ -22,6 +22,14 @@ registerBlockType( 'laterpay/contribution', {
 		campaignName: {
 			type: 'string', // Name of Campaign.
 		},
+		dialogHeader: {
+			type: 'string',
+			default: __( 'Support the author', 'laterpay' )
```

Comment:
```suggestion
			default: __( 'Support the author', 'laterpay' ),
```
Missing trailing comma

Reply:
Addressed.